index.js

```diff
@@ -58,6 +60,46 @@ app.post("/github_push_webhook/:id", async (req, res) => {
   }
 });
 
+async function post_discord_url_to_db() {
+  try {
+    const data_2 = await pool.query(
+      `insert into github_discord_url (discord_url) values ($1)`,
+      [db_url]
+    );
+  } catch (err) {
+    console.log(err.message);
+  }
+}
+
+async function get_id_by_discord_url() {
+  try {
+    const data_3 = await pool.query(
+      `select id from github_discord_url where discord_url = $1`,
+      [db_url]
+    );
+    const matching_id = data_3.rows[0];
+    return matching_id["id"];
```

Comment:
return entire row - don't return single values

Reply:
But I need single values only, what's the use of returning the entire row

Reply:
Also I'm selecting the id only from the database

Reply:
+ if I return the entire row then how could I access the id in the function which it is used in